src/Tizen.NUI/src/public/BaseComponents/FlexContainer.cs

```diff
@@ -138,6 +139,7 @@ public class FlexContainer : View
         /// Calling member functions with an uninitialized handle is not allowed.
         /// </summary>
         /// <since_tizen> 3 </since_tizen>
+        [Obsolete("Deprecated in API8, will be removed in API10. Please use FlexLayout instead!")]
```

Comment:
Great! (b^_^)b
Every time doing deprecated ACR, the Obsolete attribute should be added. Thank you.